actorguidelines.md

```diff
@@ -0,0 +1,86 @@
+# Actor Guidelines
+
+## Repository Structure
+
+Currently existing actors are stored in a common repository on Github: https://github.com/leapp-to/leapp-actors
+
+Such repository contains actors and schemas used by Leapp project. Actors stored on such repository should be stored following a directory structure based on their functionalities. As an example of such directories we have:
+* Common: Contains actors that provide functionalities that are used by a group of other actors.
+* Containerization: Contains actors specific for all types of migration to containers.
+* Containerization/Common: Contains actors with common functionalities used by other containerization actors.
+* Upgrade: Contains actors specific for all types of system-wide OS upgrades or migrations to the new OS version.
+* Upgrade/Common: Contains actors with common functionalities used by other upgrade actors.
+
+These is a structure that supports all existing use cases but, if necessary, it can be expanded in the future.
+
+
+## Actor Structure
+
+Each actor should be stored in a single directory with name based on the actor name (respecting naming conventions described below). An actor is composed of the following files:
+
+**_actor.yaml**: YAML file containing actor description used by Snactor library. This file will define used input/output channels and point to the script that will be executed by such actor. Even if not mandatory, a description should be provided in such file, in a way that developers can use it to understand purpose of an actor and how it is expected to work
+
+**script.ext**: Script file used by actor. A common practice is naming such script based on actor name or use a common name between all actors, like **exec**. Another one is writing such script using Python or Bash script.
```

Comment:
Just think, that **exec** has same name as the `exec` command so different name would be probably better. But I am not against that.